doc/dynamo0p3.rst

```diff
@@ -1086,18 +1101,32 @@ copy_scaled_field
 
 Multiplies a field by a scalar and stores the result in a second field: ::
   
-  field2(:) = value * field1(:)
+   field2(:) = value * field1(:)
 
 where:
 
 * real(r_def), intent(in) :: *value*
 * type(field_type), intent(in) :: *field1*
 * type(field_type), intent(out) :: *field2*
 
-divide_field
-++++++++++++
+divide_fields
```

Comment:
I didn't change that particular name, divide_fields is field3 = field1/field2 as before. I changed divide_field to inc_divide_field because it's writing to the same field (field1 = field1/field2).